app-android/app/src/main/java/de/tutao/tutanota/AndroidFileFacade.kt

```diff
@@ -331,7 +331,19 @@ class AndroidFileFacade(
 
 	@Throws(IOException::class)
 	override suspend fun readDataFile(filePath: String): DataFile? {
-		throw Error("FileFacade.readDataFile should not be used in Android")
+		val uri = Uri.parse(filePath)
```

Comment:
Sorry, didn't get that. Do you mean inside of our folders (encrypted/decrypted)? I think that it's fine to read arbitrary files since the user is giving us the path, I don't believe that we'll have problems to read from directories outside /temp

Reply:
We should restrict it to out encrypted/decrypted temp directories, yes. We can try to allow content provider URLs too but I think we anyway copy the files to our temp directories when we receive them, do we not?